### PR TITLE
monitoring: add detailed descriptions for each metric

### PR DESCRIPTION
**Pull request by PettitWesley**
PLEASE HELP CHECK IF MY DESCRIPTIONS ARE CORRECT, ESPECIALLY FOR THE STORAGE METRICS.

Signed-off-by: Wesley Pettit <wppttt@amazon.com>

### CODE REVIEW

**Comment by edsiper** [comment]
cc: @lecaros

**Review by DrewZhang13 — COMMENTED** [review]
There are several blank rows and columns in the second table, is it expected?

**Comment by PettitWesley** [comment]
@DrewZhang13 that seems to be something weird the markdown table generation thing I used added... thanks, I will fix it

**Comment by patrick-stephens** [comment]
This would be good to merge, are we ok now @PettitWesley @DrewZhang13 @lecaros ?

**Comment by PettitWesley** [comment]
@patrick-stephens @lecaros I want someone outside of my own team (So not Drew) to validate that this is all correct (with an approval) before we merge it.

**Review by lecaros — CHANGES_REQUESTED** [review]

**Comment by lecaros** [comment]
Hi everyone, sorry I didn't send the review. I just left it there pending. Besides those 2 changes (fluentbit_output_proc_records_total and fluentbit_output_proc_bytes_total), the docs are great!

File: administration/monitoring.md [hunk 1]
```diff
@@ -93,6 +93,51 @@ Fluent Bit aims to expose useful interfaces for monitoring, as of Fluent Bit v0.
...
+| fluentbit_output_retried_records_total | name: the name or alias for the input instance  | The number of log records that experienced a retry. Note that this is calculated at the chunk level, the count increased when an entire chunk is marked for retry. An output plugin may or may not perform multiple actions that generate many error messages when uploading a single chunk. | counter | records |
+| fluentbit_output_retries_failed_total  | name: the name or alias for the input instance  | The number of times that retries expired for a chunk. Each plugin configures a Retry_Limit which applies to chunks. Once the Retry_Limit has been reached for a chunk it is discarded and this metric is incremented.                                                                        | counter | chunks  |
```
Comment (PettitWesley) [inline administration/monitoring.md:113]: Oops I keep saying input for all of these even for the output metrics...

**Comment by PettitWesley** [comment]
@lecaros thanks. I updated it. If you can approve now then we can merge.

**Comment by lecaros** [comment]
Thanks for your contribution @PettitWesley! I've merged it.

**Pull request merged**